include/boost/beast/core/impl/flat_buffer.hpp

```diff
@@ -265,14 +265,29 @@ basic_flat_buffer<Allocator>::
 shrink_to_fit()
```

Comment:
`noexcept`?

Reply:
This would change the signature of the public interface. I didn't want to do this without concensus.

Reply:
I think we can mark it `noexcept` safely

Reply:
OK. I'll check the documentation in case it needs to mention it